setup.cfg

```diff
@@ -92,3 +92,27 @@ with-docs = 0
 
 [pycodestyle]
 max-line-length = 120
+
+# Tox global configuration options
+[tox:tox]
+# Which python versions to test for
+envlist = py37, py38, py39, py310, py311
+# Minimum version of tox required to run
+minversion = 4.0.0
+# Don't throw an error if a specific version of python is not installed
+skip_missing_interpreters = True
+# Ensure that tox uses a virtual environment to build a source distribution
+isolated_build = True
+
+# Tox configuration options for each version of python
+[testenv]
+commands =
+    # display the python version that tests are being run on
+    python -V
+    # install package requirements before running tests
+    python -m pip install -r requirements.txt
+    python -m pip install -r test-requirements.txt
+    python -m pip install -r optional-requirements.txt
+    python -m pip install -r rdocs-requirements.txt
```

Comment:
Why do we need to install the docs requirements for the tests?

Reply:
Addressed here https://github.com/blue-yonder/tsfresh/pull/994/commits/f8487373a195698f03eff374be69208f816bad42